content/storage/bind-mounts.md

```diff
@@ -266,6 +266,39 @@ $ docker container stop devtest
 $ docker container rm devtest
 ```
 
+## Recursive mounts
+
+When you bind mount a path that itself contains mounts, those submounts are
+also included in the bind mount by default. This behavior is configurable,
+using the `bind-recursive` option for `--mount`.
+
+If the bind mount is read-only, the Docker Engine makes a best-effort attempt
+at making the submounts read-only as well. This is referred to as recursive
+read-only mounts. Recursive read-only mounts require Linux kernel version 5.12
+or later. If you're running an older kernel version, submounts are
```

Comment:
Actually wondering now; what do we do for native Windows containers, @AkihiroSuda ? Do we produce an error, or do we silently ignore the options (or a warning?)

Is there an equivalent of submounts on Windows? (and are they inherited?) 🤔

Reply:
Likely to be ignored

Reply:
Yeah; wondering if Windows in any form inherits nested mounts. I can try asking around for someone to try.

If it doesn't, then ignoring is probably fine (we could print a warning in some cases I guess, but more complicated on `docker run`, because we can't mix those with the container's own output)